feature-gate/Dockerfile.tilt

```diff
@@ -1,11 +1,11 @@
 # Create minimal deployment
 # with just the service binary included
 
-FROM debian:buster-slim
+FROM ubuntu:hirsute
 ENV DEBIAN_FRONTEND=noninteractive
 RUN apt-get update -q \
     && apt-get install -y -q "libssl1.1" "libpq-dev" \
     && rm -rf /var/lib/apt/lists/*
 COPY feature-gate/target/debug/feature-gate /usr/bin/feature-gate
+COPY feature-gate/config.toml /etc/architus/config.toml
 ENV RUST_BACKTRACE=1
-ENTRYPOINT ["/usr/bin/feature-gate"]
```

Comment:
This is because we override the entrypoint in the Tiltfile (since the docker hot-reload stuff does weird stuff where it actually creates a new Docker image based on this one with the hot-reload magic, so this entrypoint is ignored anyways).